huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -1151,6 +1154,25 @@ func resourceInstanceSecGroupIdsV1(client *golangsdk.ServiceClient, d *schema.Re
 	return secgroups, nil
 }
 
+func getOpSvcUserID(d *schema.ResourceData, config *config.Config) string {
+	if v, ok := d.GetOk("user_id"); ok {
+		return v.(string)
+	}
+	return config.UserID
+}
+
+func validateComputeInstanceConfig(d *schema.ResourceData, config *config.Config) error {
+	_, useSSH := d.GetOk("key_pair")
```

Comment:
maybe isSSH is better?
Rule: For boolean args, we use `is`, `has`, `should`, `can` (non-meaning expression) as prefix definitions.

Reply:
replaced with `hasSSH`